contents/handbook/small-teams/monitoring/index.mdx

```diff
@@ -1,30 +1,37 @@
 ---
-title: Monitoring Team
+title: Team Investigation
 sidebar: Handbook
 showTitle: true
 hideAnchor: false
 ---
 
-import Mission from "./mission.mdx"
-import Objectives from "./objectives.mdx"
-
 <TeamMembers />
 
 ## Mission
 
-<Mission />
+We help customers understand _"the why"_ behind data and metrics tracked elsewhere in PostHog. Our tools help customers re-create and analyze the experience of individual users.
+
+We build the heads up display for your product
```

Comment:
```suggestion
We help customers understand _"the why"_ behind data and metrics in PostHog. Our tools help customers re-create, analyze, and understand the experience of individual users.

We build the [heads-up](https://en.wikipedia.org/wiki/Head-up_display) display for your product
```

Reply:
Ah, I missed the change in the larger thread, sorry